gef-ui/server/controllers/submit-to-ukef/index.js

```diff
@@ -59,14 +63,23 @@ const createSubmissionToUkef = async (req, res) => {
       comments.push(commentObj);
       application.comments = comments;
     }
+
+    /**
+     * Change the application submission type to MIN from MIA,
+     * upon second submission. This is to ensure correct confirmation
+     * message is shown on the NJK template else isNotice() will return false
+     */
+    const isDealNotice = application.ukefDecisionAccepted ? true : isNotice(application.submissionType);
+
     // Always update with the latest checkers details.
     application.checkerId = user._id;
     await api.updateApplication(dealId, application);
     await api.setApplicationStatus(dealId, CONSTANTS.DEAL_STATUS.SUBMITTED_TO_UKEF);
     // TODO: DTFS2-4706 - add a route and redirect instead of rendering?
     return res.render('partials/submit-to-ukef-confirmation.njk', {
       submissionType: application.submissionType,
-      isNotice: isNotice(application.submissionType),
+      isNotice: isDealNotice,
+      ukefDecisionAccepted,
```

Comment:
Can we update the test for new property and condition? `isDealNotice` could also be extracted and have it's own test